build.gradle

```diff
@@ -148,8 +148,12 @@ test {
 	systemProperty 'nio_test_file', "delete-nio-testng."
 	systemProperty 'sage.testing', "true"
 
-    workingDir = file("$sageBuildDir/sagetv_test/");
-    workingDir.mkdirs();
+    workingDir = file("$sageBuildDir/sagetv_test/")
```

Comment:
Indentation on these 2 lines looks off.

Reply:
it appears that the `build.gradle` file was a mix to tabs and spaces.  I reformatted to use spaces, and updated `.editorconfig` to ensure consistency moving forward.